packages/shared/src/models/IRecipe.ts

```diff
@@ -5,6 +5,7 @@ export interface Recipe {
   description: string;
   icon?: string;
   repository: string;
+  ref: string;
```

Comment:
I think this should be optional. You could add some test to assert it works with an undefined ref